### ISSUE TITLE
FP_SERVER_ERROR

### ISSUE THREAD

**Issue by aphoenix01** [EMCECS/ecs-sync#62]
I am trying to sync data between EMC Centera and EMC ECS/Centera but keep getting the following error..

Failed to store object: CAS Error -10005/FP_SERVER_ERROR: Unknown Error

Can anybody confirm what this error code means.

**Comment by holgerjakob** [comment]
Hi 
The error is a pretty generic one. Are you able to run a JCenteraVerify using the pea file of the ECS user?
Sometimes this is related to not having the default bucket set.

Best regards, Holger

**Comment by aphoenix01** [comment]
Hi Holger,

Thanks for responding.

I have split the source clip list into 4 and have been running each on separate Sync Hosts, for 3 it has been fine, but this last one has been very problematic and I wasn’t sure of the root cause. I then tried the same cliplist on all the other Sync Hosts that had worked and get the same issue.

I have tried using jCenterVerify but I keep getting the following errors relating to FPLibrary. I have updated the LD_LIBRARY_PATH to point to the location of these files, but get exactly the same problem.

[cid:image001.png@01D5AABE.910093E0]

I am further hampered by a lack of experience using linux, so if I ask stupid questions that will be why. 😊 But also there may be an easy answer to the problem.

My concern is there are issues with the objects on Centera preventing me from copying them, have seen this before on Centeras that have been around for a while and saved objects in an older format, not entirely sure how to verify this.


Regards

Andrew
